src/gui/ar/areaTargetScanner.js

```diff
@@ -310,13 +336,19 @@ createNameSpace("realityEditor.gui.ar.areaTargetScanner");
         showLoadingDialog('Generating Dataset...', 'Please wait. Converting scan into AR target files.');
     }
 
-    function createPendingWorldObject() {
-        console.log('createPendingWorldObject()');
+    function createPendingWorldObject(serverIp) {
+        console.log('createPendingWorldObject()', detectedServers);
 
         // TODO: first create a new object and post it to the server
-        let randomServerIP = Object.keys(detectedServers)[0]; // this is guaranteed to have at least one entry if we get here
+        let serverIps = Object.keys(detectedServers);
+        if (!serverIps.includes(serverIp)) {
+            serverIp = serverIps.filter(detectedServer => {
+                return !['127.0.0.1', 'localhost'].includes(detectedServer);
```

Comment:
Not sure if this also needs the 10.10.10.10/20? But also shouldn't be too necessary since that's just for our demo setup.